internal/test/e2e/e2e_test.go

```diff
@@ -48,16 +49,10 @@ var (
 	fUseKind           = flag.Bool("kind", false, "Creates a Kind cluster to run the tests against.")
 	fOutputDirectory   = flag.String("output-directory", "/tmp/etcd-e2e", "The absolute path to a directory where E2E results logs will be saved.")
 	fKindClusterName   = flag.String("kind-cluster-name", "etcd-e2e", "The name of the Kind cluster to use or create")
+	fControllerImage   = flag.String("controller-image", "", "The name of the controller image")
 	fUseCurrentContext = flag.Bool("current-context", false, "Runs the tests against the current Kubernetes context, the path to kube config defaults to ~/.kube/config, unless overridden by the KUBECONFIG environment variable.")
 	fRepoRoot          = flag.String("repo-root", "", "The absolute path to the root of the etcd-cluster-operator git repository.")
 	fCleanup           = flag.Bool("cleanup", true, "Tears down the Kind cluster once the test is finished.")
-
-	etcdConfig = etcd.Config{
-		Endpoints: []string{"http://127.0.0.1:2379"},
-		Transport: etcd.DefaultTransport,
-		// set timeout per request to fail fast when the target endpoint is unavailable
-		HeaderTimeoutPerRequest: time.Second,
-	}
```

Comment:
We can't rely on being able to connect to the etcd cluster inside K8s. 
I've replaced this with an in-cluster execution of etcdctl (as used in other tests), which should allow this test to pass in any k8s cluster.